en/security.rst

```diff
@@ -10,6 +10,53 @@ This chapter describes the CUBRID security features. CUBRID provides Packet Encr
 Packet Encryption
 =================
 
+The need for packet encryption
```

Comment:
Requirements of secure communication

Reply:
Thank you, i will change text as your recommend.